### PR TITLE
Milestone 1: Basic project structure

### PR DESCRIPTION
**Pull request by verissimohenry**
Setting up some basic structure and tooling for a calculator for Math-magicians Inc using create-react-app, using a tool built by developers at Facebook to help you build React applications

### CODE REVIEW

**Review by eliasecasta — CHANGES_REQUESTED** [review]
## Status:  Invalid 🔴 

Hi, @verissimohenry, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that has no descriptive enough commit messages, please correct this issue with git rebase. Specifically your "modify app" commits, please change that text to something descriptive of what did you do on those commits, please.
- [x] For your next milestones please give a more descriptive naming to your feature branch too, for this one, instead of feature it would be better to name it like feature/project-setup or feature/project-initialization, feature/milestone-1, or anything that is descriptive enough. Please don't make this change in the current pull request because this review will get lost, just take it in mind for your next feature branches.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,117 @@
-# calculator-react
\ No newline at end of file
+# Calculator-App
+[Live Link]()
```
Comment (rindrajosia) [inline README.md]: - [x] [REQUIRED]app should be deployed on Heroku(or other hosts like Netlify, Firebase, etc) and the live link is included at the README file.
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```
Comment (rindrajosia) [inline package.json:5]: - [x] [REQUIRED] `big.js` should be added to package.json. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/big.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './styles/index.css';
+import App from './components/App';
+// import PropTypes from 'prop-types';
+// import Big from 'big.js';
```
Comment (rindrajosia) [inline src/index.js]: - [x] [REQUIRED] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] Files not related to the project should not be committed and added to .gitignore (e.g. .vscode) 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [x] I got an error when open your APP on my local. Please fix it:



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- to make your commit more descriptive.
- to give a more descriptive naming to your feature branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by bellom — APPROVED** [review]
## Hi @verissimohenry  👏  

**Great work making all the necessary corrections from the previous review.**

As the previous reviewer mentioned 👇 
Files not related to the project should not be committed and added to .gitignore (e.g. .vscode)

Make sure to add `.vscode` to `gitignore`, else the next CRR won't get approved.

I trust you can do this before merging to dev, 👀  on you.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**